tests/console/console.go

```diff
@@ -128,6 +128,20 @@ func RunCommand(vmi *v1.VirtualMachineInstance, command string, timeout time.Dur
 	return nil
 }
 
+// RunSafeCommandAndSucceed runs the command line from `command connecting to an already logged in console at vmi
```

Comment:
the name in comment is inconsistent with the real one (that's always a problem with comments!)

also, the name `RunSafeCommandSuccessfully` is not clear to me. I don't know why it is "safe" nor "successful". 
How do you feel about `RunConsoleCommand`? `RunAndCheckShellCommand`?

Reply:
We have documented what the command means in the godoc and we express what "Safe" means and also we know what Successfully means implcitly, so I think this is ok.

I will change the commit comments though, thanks for review.

Reply:
I may well be mistaken, as I'm not familiar with the function and its usages. I'm not holding this PR. From elsewhere in this file it seems that  "Safe" means "ValidatedSend". Is that so? Does the standard library or another library you value have `Successfully` is a function name? `console.RunConsoleCommand` is indeed bad.

BTW should the backtick before "command" be closed?

Reply:
> I may well be mistaken, as I'm not familiar with the function and its usages. I'm not holding this PR. From elsewhere in this file it seems that "Safe" means "ValidatedSend". Is that so? Does the standard library or another library you value have `Successfully` is a function name? `console.RunConsoleCommand` is indeed bad.
> 
> BTW should the backtick before "command" be closed?

I associate "success" while executing a command with return code 0, which is something asserted within that function. That was the reason why I insisted in adding that to the function name. 

Can't venture into the "safe" part.

Reply:
What about `console.RunCommandAndValidateSend` ?

Reply:
Reading this again, I'm forced to agree this method's name is confusing. 

I don't know what the safe regards to.